tst/EnergyPlus/unit/HeatBalFiniteDiffManager.unit.cc

```diff
@@ -294,103 +294,108 @@ TEST_F(EnergyPlusFixture, HeatBalFiniteDiffManager_adjustPropertiesForPhaseChang
-        "   368, !- Density{ kg / m3 }",
-        "   590.000000000002, !- Specific Heat{ J / kg - K }",
-        "   0.9, !- Thermal Absorptance",
-        "   0.3, !- Solar Absorptance",
-        "   0.3;                     !- Visible Absorptance",
-
-        "Material,",
-        "   M11 100mm lightweight concrete, !- Name",
-        "   MediumRough, !- Roughness",
-        "   0.1016, !- Thickness{ m }",
-        "   0.53, !- Conductivity{ W / m - K }",
-        "   1280, !- Density{ kg / m3 }",
-        "   840.000000000002, !- Specific Heat{ J / kg - K }",
-        "   0.9, !- Thermal Absorptance",
-        "   0.5, !- Solar Absorptance",
-        "   0.5;                     !- Visible Absorptance",
-
-        "Construction,",
-        "  ExtSlabCarpet 4in ClimateZone 1 - 8, !- Name",
-        "  MAT - CC05 4 HW CONCRETE, !- Outside Layer",
-        "  CP02 CARPET PAD;         !- Layer 2",
-        "Construction,",
-        "   Interior Floor, !- Name",
-        "   F16 Acoustic tile, !- Outside Layer",
-        "   F05 Ceiling air space resistance, !- Layer 2",
-        "   M11 100mm lightweight concrete;  !- Layer 3",
-        "Construction:AirBoundary,",
-        "   Air Wall_ConstructionAirBoundary,  !- Name",
-        "   None,                    !- Air Exchange Method",
-        "   0;                       !- Simple Mixing Air Changes per Hour {1 / hr}",
-        "Output:Constructions,",
-        "Constructions;",
-        "Output:Constructions,",
-        "Materials;",
-    });
-
-    ASSERT_TRUE(process_idf(idf_objects));
-
-    ErrorsFound = false;
-    Material::GetMaterialData(*state, ErrorsFound); // read material data
-    EXPECT_FALSE(ErrorsFound);                      // expect no errors
-
-    ErrorsFound = false;
-    HeatBalanceManager::GetConstructData(*state, ErrorsFound); // read construction data
-    EXPECT_FALSE(ErrorsFound);                                 // expect no errors
-
-    // allocate properties for construction objects when it is used or not for building surfaces in the model
-
-    state->dataConstruction->Construct(1).IsUsed = false;
-    state->dataConstruction->Construct(2).IsUsed = true;
-    state->dataConstruction->Construct(3).IsUsed = true;
-
-    // call the function for initialization of finite difference calculation
-    InitialInitHeatBalFiniteDiff(*state);
-    auto &ConstructFD = state->dataHeatBalFiniteDiffMgr->ConstructFD;
-    // check the values are correct
-    EXPECT_EQ(0, ConstructFD(1).Name.size());
-    EXPECT_EQ(3, ConstructFD(2).Name.size());
-    EXPECT_EQ(0, ConstructFD(3).Name.size());
-    EXPECT_EQ("F16 ACOUSTIC TILE", ConstructFD(2).Name(1));
-    EXPECT_EQ("F05 CEILING AIR SPACE RESISTANCE", ConstructFD(2).Name(2));
-    EXPECT_EQ("M11 100MM LIGHTWEIGHT CONCRETE", ConstructFD(2).Name(3));
-
-    // deallocate
-    ConstructFD.deallocate();
-}
+// I'm not sure how this test was intended to work, there doesn't appear to be anything setting the constructions
```

Comment:
There was a test here that was disabled, and hadn't been run in a long time.  Since ctest is more aggressively reporting out disabled tests, I commented it out.  Happy to get it back in if someone wants to spend time investigating the failure.